### PR TITLE
Multi bracket validation

### PR DESCRIPTION
**Pull request by paronicholas**
- [x] Top-level README “Table of Contents” is updated
- [x] Feature tasks for this challenge are completed
- [x] Unit tests written and passing
    - [x] “Happy Path” - Expected outcome
    - [x] Expected failure
    - [x] Edge Case (if applicable/obvious)
- [x] README for this challenge is complete
    - [x] Summary, Description, Approach & Efficiency, Solution, Resources
    - [x] Link to code
    - [x] Link to tests

### CODE REVIEW

File: code401challenges/src/main/java/utilities/MultiBracketValidation.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package utilities;
+
+import stacksAndQueues.Stack;
+
+public class MultiBracketValidation {
+  public static boolean multiBracketValidation(String input) {
+    Stack<Character> checkStack = new Stack<>();
+    char[] chars = input.toCharArray();
```
Comment (evanslaton) [inline code401challenges/src/main/java/utilities/MultiBracketValidation.java]: Instead of converting the string into an array (and taking up extra memory) you can iterate over a string in Java by using the `String` method `.charAt()`:

File: code401challenges/src/main/java/utilities/MultiBracketValidation.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    for (int i = 0; i < chars.length; i++) {
```
Comment (evanslaton) [inline code401challenges/src/main/java/utilities/MultiBracketValidation.java]: And then you use the `.length()` method in the for loop conditional. Be sure to recognize that `.length` is a property of an array while `.length()` is a method on string in Java
